cioos_data_transform/cioos_data_transform/utils/odf.py

```diff
@@ -214,15 +218,32 @@ def _find_previous_key(key_list, key):
     for flag_column, data_column in flag_dict.items():
         if data_column in metadata:
             if 'name' in metadata[data_column]:
-                metadata[flag_column].update({'name': flag_prefix + metadata[data_column]['name']})
+                metadata[flag_column].update({'long_name': flag_prefix + metadata[data_column]['name']})
             else:
-                metadata[flag_column].update({'name': flag_prefix + data_column})
+                metadata[flag_column].update({'long_name': flag_prefix + data_column})
         # TODO improve flag parameters default documentation
         #  - add ancillary_variables to the associated variable attributes
         #       http://cfconventions.org/cf-conventions/cf-conventions.html#ancillary-data
         #  - add flag_values, flag_masks and flag_meanings to flag attributes
         #       http://cfconventions.org/cf-conventions/cf-conventions.html#flags
 
+    # null_values / fill_values
+    # Deal with fill value
+    for key, var in metadata.items():
```

Comment:
this could be separated out into a different function with comment explaining the function's input and output